css/style.css

```diff
@@ -0,0 +1,2929 @@
+}
+
+.button__btn-text {
+  margin-right: 5px;
+}
+
+.bottom__info-img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+@media (max-width: 1400px) {
+  .top__img {
+      margin-right: 0;
+  }
+
+  .bottom__info-img {
+      display: none;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 520px 142px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 400px 150px;
+      background-size: 900px 500px ;
+    }
+}
+
+@media (max-width: 1350px) {
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 420px 150px;
+      background-size: 900px 500px ;
+  }
+
+}
+
+@media (max-width: 1300px) {
+  .container {
+      max-width: 1300px;
+  }
+
+  .menu {
+      margin-left: 0;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 1300px 467px;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 700px 150px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 420px 150px;
+      background-size: 800px 500px ;
+  }
+
+}
+
+@media (max-width: 1200px) {
+  .container {
+      max-width: 100%;
+  }
+
+  .header__menu-link {
+      display: none;
+  }
+
+  .menu__link {
+      display: none;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 1200px 467px;
+  }
+
+  .top__wrapper-item {
+      margin-right: 80px;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 500px 150px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 415px 150px;
+      background-size: 700px 400px ;
+  }
+
+}
+
+@media (max-width: 1100px) {
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 1150px 467px;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 500px 130px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 430px 200px;
+      background-size: 600px 350px ;
+  }
+
+}
+
+@media (max-width: 1024px) {
+  .container {
+      max-width: 100%;
+  }
+
+  .header__menu-link {
+      display: none;
+  }
+
+ .menu__link {
+      display: none;
+  }
+
+  .top__wrapper-charset {
+      max-width: 100%;
+      height: auto;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)